react/features/recording/googleApi/googleApiFactory.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import { hasElectronIntegration } from './functions';
+import googleElectronClient from './googleElectronClient';
+import googleWebClient from './googleWebClient';
+
+declare var JitsiMeetElectron: Object;
+
+export default {
+    /**
+     * Obtains the Google client for the current platform.
+     *
+     * @returns {Object} The client to use for interacting with the Google API.
+     */
+    getClient() {
+        if (hasElectronIntegration()) {
```

Comment:
discussed offline: have this do the test for electron and either return the electron client, the web client, or nothing.  then you can inject the result of this into props and use that as the cue as to whether or not the integration should be shown.